docs/docs.json

```diff
@@ -22,7 +22,7 @@
         "groups": [
           {
             "group": "Getting Started",
-            "pages": ["index", "quickstart"]
+            "pages": ["index", "cli/installation", "quickstart"]
```

Comment:
Clicking this link feels a bit jarring as the whole nav changes. An alternative could be to move the CLI reference as a dropdown menus and keep a single nav.

Reply:
Cool I'll resolve this one